api-reference/10 UI Widgets/dxDataGrid/1 Configuration/remoteOperations/summary.md

```diff
@@ -8,4 +8,4 @@ default: false
 Specifies whether or not [summaries](/concepts/05%20Widgets/DataGrid/65%20Summaries/10%20Total%20Summary '/Documentation/Guide/Widgets/DataGrid/Summaries/Total_Summary/') calculation must be performed on the server.
 
 ---
-[note] When summary calculation is remote, the widget does not support [custom summary calculation](/concepts/05%20Widgets/DataGrid/65%20Summaries/07%20Custom%20Aggregate%20Function '/Documentation/Guide/Widgets/DataGrid/Summaries/Custom_Aggregate_Function/').
\ No newline at end of file
+[note]When [custom summary calculation](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/summary/#calculateCustomSummary) is performed on a server, the [allowExportSelectedData](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/export/#allowExportSelectedData) option is disabled.
```

Comment:
Again, "option is disabled." - "option is not supported" look better for me.

Reply:
Replied above

Reply:
Do you mean we should just remove this note?